packages/plugins/src/hardhat/simulate.ts

```diff
@@ -192,10 +192,14 @@ export const simulate = async (
     throw new Error(errorMessage)
   }
 
-  const receipts = JSON.parse(stdout).receipts
-  const batches = JSON.parse(stdout).batches.map(toSphinxLeafWithProofArray)
-
-  return { receipts, batches }
+  try {
+    const receipts = JSON.parse(stdout).receipts
+    return { receipts }
+  } catch (e) {
+    console.log(stdout)
+    console.error(stderr)
+    process.exit(1)
+  }
```

Comment:
I've encountered a few situations where an error occurs that causes stdout not to be valid JSON. It will then fail to be parsed, and the process will crash. This is annoying because the real error that caused the problem will not be logged at all. So this just catches that case and logs `stdout` and `stderr` so we can access whatever the error is.

Reply:
Sounds good. nit: it'd be nice if this explanation is documented in the code